### PR TITLE
Aggregate the user mail alerts to send aggregated mail

### PR DESCRIPTION
**Pull request by athiruma**
1. Aggregate the policy alerts
2. Changed the logger_time_stamp decorator to pass arguments
     1. When passing large data in the arguments, default printing the whole arguments data.
     2. To avoid this pass the *print_args=False* parameter to logger_time_stamp to print only argument keys

### CODE REVIEW

**Review by ebattat — COMMENTED** [review]
@athiruma , I think that we can enable/disable the logger stdout by env variable and not by passing argument for each decorator, you can change it [here](https://github.com/redhat-performance/cloud-governance/blob/c3693556c4326699442556ff0dfeea40bf2f9764/cloud_governance/common/logger/init_logger.py#L12)

File: cloud_governance/policy/aws/unused_nat_gateway.py [hunk 1]
```diff
@@ -25,6 +25,8 @@ def __check_cloud_watch_logs(self, resource_id: str, days: int = UNUSED_DAYS):
         :param days:
         :return:
         """
+        if days == 0:
+            days = 1
```
Comment (ebattat) [inline cloud_governance/policy/aws/unused_nat_gateway.py:29]: Is it ok to revert it back to 1 ?

Comment (athiruma) [reply cloud_governance/policy/aws/unused_nat_gateway.py:29]: we need to always check for the last one day.

Comment (athiruma) [reply cloud_governance/policy/aws/unused_nat_gateway.py:29]: we need to always check for the last one day.

**Comment by athiruma** [comment]
> @athiruma , I think that we can enable/disable the logger stdout by env variable and not by passing argument for each decorator, you can change it [here](https://github.com/redhat-performance/cloud-governance/blob/c3693556c4326699442556ff0dfeea40bf2f9764/cloud_governance/common/logger/init_logger.py#L12)

I need a logger for some methods, but I don't need the values to print  stdout

File: cloud_governance/common/logger/logger_time_stamp.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 from cloud_governance.common.logger.init_logger import logger
 
+
```
Comment (ebattat) [inline cloud_governance/common/logger/logger_time_stamp.py:7]: Please remove extra space

File: cloud_governance/common/clouds/aws/ec2/ec2_operations.py [hunk 3]
```diff
@@ -516,3 +516,13 @@ def get_tag_value_from_tags(self, tags: list, tag_name: str, default_value: any
...
+    def get_active_regions(self):
+        """
+        This method return active regions in aws account
+        :return:
+        """
+        responses = self.ec2_client.describe_regions()['Regions']
+        active_regions = []
+        for region in responses:
+            active_regions.append(region.get('RegionName'))
+        return active_regions
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/ec2/ec2_operations.py:528]: That will return all active **empty** regions, is that what we want ?

**Pull request merged**